content/tracing/setup/python.md

```diff
@@ -3,8 +3,23 @@ title: Tracing Python Applications
 kind: Documentation
 aliases:
 - /tracing/python/
+- /tracing/languages/python/
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-py"
+  tag: "Github"
+  text: Source code
+- link: "http://pypi.datadoghq.com/trace/docs/"
+  tag: "Pypi"
+  text: APM (Tracing)
+- link: "/tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
 ---
 
+<div class="alert alert-info">
+For Python applciations, note that tracing is disabled when your application is launched in <b>DEBUG</b> mode. Find more <a href="http://pypi.datadoghq.com/trace/docs/#module-ddtrace.contrib.django">here</a>.
```

Comment:
I'd use `<code>DEBUG</code>` instead of `<b>DEBUG</b>`.

Reply:
👌